packages/react/src/components/Notification/Notification.js

```diff
@@ -266,25 +267,33 @@ export function ToastNotification({
     [`${prefix}--toast-notification--${kind}`]: kind,
   });
 
+  const handleClose = useCallback(
+    (evt) => {
+      if (!onClose || onClose(evt) !== false) {
```

Comment:
For the `onClose(evt) !== false` branch, would we more so want people to be able to configure visibility of the close button if they aren't able to dismiss/close the notification?

Reply:
this is for the visibility of the notification itself, similar to how the visibility of modals is controlled rather than the visibility of the modal close button

Reply:
@emyarod right, the way I interpreted it was that if the `onClose` prop is called and returns `false` then it will prevent the close from happening. Is that correct?

If so, then I was just asking that if the notification is unable to be dismissed (the `onClose` prop would return `false`) it seems like there would need to be some indicator of that. I think from my comment that was hiding the button if someone couldn't dismiss the notification but it could be something else, for sure.

Reply:
wouldn't this be left up to the consumer to resolve? similar to how we provide this API in our modals